addons/event/static/src/template_reference_field/field_event_mail_template_many2one.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<templates xml:space="preserve">
+
+    <t t-name="event.mail_template_many2one_field" t-inherit="web.Many2OneField" t-inherit-mode="primary">
+        <!--always display without open link when readonly-->
+        <xpath expr="//t[@t-if='props.readonly']/t[@t-if='!props.canOpen']" position="attributes">
```

Comment:
Isn't it easier and more straightforward to manipulate "canOpen" then? Be it through field options on the form view or if not possible through your JS override?